src/v/cluster/rm_stm.cc

```diff
@@ -43,14 +43,37 @@ static bool is_sequence(int32_t last_seq, int32_t next_seq) {
            || (next_seq == 0 && last_seq == std::numeric_limits<int32_t>::max());
 }
 
-static model::record_batch
-make_fence_batch(int8_t version, model::producer_identity pid) {
+static model::record_batch make_fence_batch(model::producer_identity pid) {
```

Comment:
this commit will not compile as the call site isn't changed.

Reply:
squashed it with the next one